x-pack/test_serverless/api_integration/test_suites/common/saved_objects_management/scroll_count.ts

```diff
@@ -0,0 +1,194 @@
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+// import { SavedObjectWithMetadata } from '@kbn/saved-objects-management-plugin/common';
+import { FtrProviderContext } from '../../../ftr_provider_context';
+import { RoleCredentials } from '../../../../shared/services';
+
+const apiUrl = '/api/kibana/management/saved_objects/scroll/counts';
+const defaultTypes = ['visualization', 'index-pattern', 'search', 'dashboard'];
+
+export default function ({ getService }: FtrProviderContext) {
+  const svlCommonApi = getService('svlCommonApi');
+  const svlUserManager = getService('svlUserManager');
+  const supertest = getService('supertest');
+  const kibanaServer = getService('kibanaServer');
+  let roleAuthc: RoleCredentials;
+
+  describe('scroll_count', () => {
+    before(async () => {
+      roleAuthc = await svlUserManager.createApiKeyForRole('admin');
+    });
+
+    after(async () => {
+      await svlUserManager.invalidateApiKeyForRole(roleAuthc);
+    });
+    describe('scroll_count with more than 10k objects', () => {
+      const importVisualizations = async ({
+        startIdx = 1,
+        endIdx,
+      }: {
+        startIdx?: number;
+        endIdx: number;
+      }) => {
+        const fileChunks: string[] = [];
+        for (let i = startIdx; i <= endIdx; i++) {
+          const id = `test-vis-${i}`;
+          fileChunks.push(
+            JSON.stringify({
+              type: 'visualization',
+              id,
+              attributes: {
+                title: `My visualization (${i})`,
+                uiStateJSON: '{}',
+                visState: '{}',
+              },
+              references: [],
+            })
+          );
+        }
+
+        await supertest
+          .post(`/api/saved_objects/_import`)
+          .set(svlCommonApi.getInternalRequestHeader())
+          .set(roleAuthc.apiKeyHeader)
+          .attach('file', Buffer.from(fileChunks.join('\n'), 'utf8'), 'export.ndjson')
+          .expect(200);
+      };
+
+      const deleteVisualizations = async ({
+        startIdx = 1,
+        endIdx,
+      }: {
+        startIdx?: number;
+        endIdx: number;
+      }) => {
+        const objsToDelete: any[] = [];
+        for (let i = startIdx; i <= endIdx; i++) {
+          const id = `test-vis-${i}`;
+          objsToDelete.push({ type: 'visualization', id });
+        }
+        await kibanaServer.savedObjects.bulkDelete({ objects: objsToDelete });
+      };
+
+      before(async () => {
+        await kibanaServer.savedObjects.cleanStandardList();
+        await importVisualizations({ startIdx: 1, endIdx: 6000 });
+        await importVisualizations({ startIdx: 6001, endIdx: 12000 });
+      });
+
+      after(async () => {
+        // kibanaServer.savedObjects.cleanStandardList({}); times out for 12000 items
+        await deleteVisualizations({ startIdx: 1, endIdx: 3000 });
```

Comment:
q: is this a zero-based index?

Reply:
no, it's a natural number (positive starting from 1)